test/functional/test_runner.py

```diff
@@ -52,6 +52,8 @@
 TEST_EXIT_PASSED = 0
 TEST_EXIT_SKIPPED = 77
 
+TRAVIS_TIMEOUT_DURATION = 1200 # 20 minutes represented in seconds
```

Comment:
Keep `20 * 60` expression?

Reply:
@promag I'm fine either way.  Wasn't sure if people would read it as `twenty * sixty seconds` or not.  I can make that change later today unless anyone prefers `1200`. Thanks!